nft_ingester/src/program_transformers/bubblegum/db.rs

```diff
@@ -96,7 +97,7 @@ where
         // sequence number.  So in this case we set at flag to force checking the tree.
         let force_chk = rows.is_empty() && change_log_event.seq > 1;
 
-        println!("Adding to backfill_items table at level {}", i - 1);
+        info!("Adding to backfill_items table at level {}", i - 1);
```

Comment:
this needs to be info?

Reply:
Could be debug. It's pretty rare and not very noisy.